adc/ade7913/ade7913-driver.c

```diff
@@ -52,6 +52,8 @@
 #define log_error(fmt, ...) do { printf(LOG_TAG COL_RED fmt COL_NORMAL "\n", ##__VA_ARGS__); } while (0)
 /* clang-format on */
 
+/* enable debugging of DMA IRQ not in-sync processing */
+#define DEBUG_NOTSYNC 1
```

Comment:
I'm not sure about committing a code with debug enabled
```suggestion
/* enable debugging of DMA IRQ not in-sync processing */
#define DEBUG_NOTSYNC 0
```

Reply:
I need this for NIL project and don't want every user to ask to enable it manually.